all-repos.yaml

```diff
@@ -219,3 +219,4 @@
 - https://github.com/sirwart/ripsecrets
 - https://github.com/bagerard/graphviz-dot-hooks
 - https://github.com/omnilib/ufmt
+- https://github.com/Qarik-Group/gitlab-codeowners-linter
```

Comment:
https://github.com/Qarik-Group/gitlab-codeowners-linter/blob/901029be1587bf9a005efe1688fb70447033b102/.pre-commit-hooks.yaml#L7-L8

- this regex should have a `$`
- `pass_filenames: false` doesn't gel nicely with the framework -- your tool should handle filenames passed as positional arguments

Reply:
I fixed the 2 points you mentioned

Reply:
the second was not fixed -- your tool should handle a variable number of positional arguments and lint those files

Reply:
That's not a valid use case, there can be [only 1 COEOWNERS file per repo](https://docs.gitlab.com/ee/user/project/code_owners.html#set-up-code-owners)

Reply:
I modified the tool, it can now handle a variable number of positional arguments and lint those files